Refs #32339 -- Allowed renderer to specify default form and formset templates. (#15622)
django/django: docs/ref/forms/renderers.txt

should return a rendered templates (as a string) or raise
:exc:`~django.template.TemplateDoesNotExist`.

.. class:: BaseRenderer

    The base class for the built-in form renderers.

    .. attribute:: template_name

        .. versionadded:: 4.1

        The name of the template to use to render the form if none is provided
        to ``render()``.

        Defaults to ``"django/forms/default.html"``, which is a proxy for
        ``"django/forms/table.html"``.

    .. method:: get_template(template_name)

>         Hook that subclasses should implement with the appropriate template
>         finding logic.

Reviewer: Maybe:
```suggestion
        Subclasses must implement this method with the appropriate template
        finding logic.
```
